pkgs/applications/radio/wfview/default.nix

```
@@ -0,0 +1,66 @@
+{ stdenv, lib, fetchFromGitLab, qcustomplot, qt5, libopus, eigen, portaudio
+, rtaudio, hidapi, udev, pulseaudio }:
+stdenv.mkDerivation {
+  name = "wfview";
```

```suggestion
  pname = "wfview";
```
`version` is missed.